run-onboarding.sh

```
@@ -278,7 +278,7 @@ function create_partner() {
     --env-var leaf-certificate="$PartnerCert" \
     --folder upload-ca-certificate \
     --folder upload-leaf-certificate \
-    -r htmlextra --reporter-htmlextra-export $(prop 'report_dir')/$partner_kc_username.html
+    -r htmlextra --reporter-htmlextra-export $(prop 'report_dir')/$partner_kc_username.html --unncessary command to test
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)